osf/utils/notifications.py

```diff
@@ -5,10 +5,6 @@
 from osf.utils.workflows import RegistrationModerationTriggers
 
 def get_email_template_context(resource):
-    provider = getattr(resource, 'provider', False)
-    assert provider, 'This resource does not fit the template'
-    assert provider.type in ['osf.preprintprovider', 'osf.registrationprovider']
-
     is_preprint = resource.provider.type == 'osf.preprintprovider'
```

Comment:
silly question, can we check resource.type instead of resource.provider.type?

Reply:
No, type here is only reffering to typed models, preprint used to be one, but is now it's own thing, separate from nodes.